libc/test/include/CMakeLists.txt

```diff
@@ -79,3 +79,13 @@ add_libc_test(
   DEPENDS
     libc.include.llvm-libc-macros.stdckdint_macros
 )
+
+add_libc_test(
```

Comment:
In addition to the C++ test to make sure that the implementation is correct, also add a light-weight C test similar to https://github.com/llvm/llvm-project/blob/main/libc/test/include/CMakeLists.txt#L49 to make sure that the macro definitions work in C.

Reply:
@lntue I'm curious if there is an example of a corresponding C test for generic functions like these macros?

Reply:
Since these function macros are defined by simply using the builtins, you could have some simple tests to make sure the macros are defined like:
```
#ifndef isfinite
#error "isfinite macro is not defined"
#else
assert(isfinite(1.0f));
assert(isfinite(1.0));
assert(isfinite(1.0L));
#endif // isfinite
```